db/migrate/20180829123352_change_milestones_to_acts_as_lesson_plan_item.rb

```diff
@@ -0,0 +1,34 @@
+class ChangeMilestonesToActsAsLessonPlanItem < ActiveRecord::Migration[5.1]
+  def up
+    ActsAsTenant.without_tenant do
```

Comment:
How long does this take? If it takes too long write the raw SQL and do something like https://github.com/Coursemology/coursemology2/blob/26b3d03d11519ef8e940b23b8991021cef1ba9b5/db/migrate/20180703023011_create_course_assessment_skills_question_assessments.rb#L17

Reply:
Pretty fast on the docker image, roughly 1-2 mins.

Reply:
yup this is nearly instantaneous when run with psql.

```
INSERT INTO course_lesson_plan_items (actable_id, actable_type, course_id, title, description, start_at, base_exp, time_bonus_exp, creator_id, updater_id, created_at, updated_at) (SELECT id, 'Course::LessonPlan::Milestone', course_id, title, description, start_at, 0, 0, creator_id, updater_id, created_at, updated_at FROM course_lesson_plan_milestones)
```